github/compliance/report.py

```diff
@@ -47,11 +47,35 @@
 _compliance_label_os_outdated = github.compliance.issue._label_os_outdated
 _compliance_label_checkmarx = github.compliance.issue._label_checkmarx
 _compliance_label_malware = github.compliance.issue._label_malware
-_compliance_label_cfg_policy_violation = github.compliance.issue._label_cfg_policy_violation
+_compliance_label_credentials_outdated = github.compliance.issue._label_outdated_credentials
+_compliance_label_no_responsible = github.compliance.issue._label_no_responsible
+_compliance_label_no_rule = github.compliance.issue._label_no_rule
+_compliance_label_no_status = github.compliance.issue._label_no_status
+_compliance_label_undefined_policy = github.compliance.issue._label_undefined_policy
 
 _ctx_label_prefix = github.compliance.issue._label_prefix_ctx
 
 
+def _friendly_label_name(label: str) -> str:
```

Comment:
why? just include this in the issue-template(s), as done for the other compliance-type-labels